### PR TITLE
Add JavaTimeModule to Jersey. Exclude Jackson from athenz bundle

### PR DESCRIPTION
**Pull request by bjorncs**

### CODE REVIEW

File: container-jersey2/src/main/scala/com/yahoo/container/servlet/jersey/JerseyServletProvider.scala [hunk 1]
```diff
@@ -96,7 +96,8 @@ class JerseyServletProvider(restApiContext: RestApiContext) extends Provider[Ser
 
   def jacksonDatatypeJdk8Provider: JacksonJaxbJsonProvider = {
     val provider = new JacksonJaxbJsonProvider()
-    provider.setMapper(new ObjectMapper().registerModule(new Jdk8Module))
+    provider.setMapper(
+      new ObjectMapper().registerModule(new Jdk8Module).registerModule(new JavaTimeModule))
```
Comment (gjoranv) [inline container-jersey2/src/main/scala/com/yahoo/container/servlet/jersey/JerseyServletProvider.scala]: A linebreak would be nice to separate the modules.

File: vespa_jersey2/pom.xml [hunk 2]
```diff
@@ -46,6 +46,10 @@
       <groupId>com.fasterxml.jackson.datatype</groupId>
       <artifactId>jackson-datatype-jdk8</artifactId>
     </dependency>
+    <dependency>
```
Comment (gjoranv) [inline vespa_jersey2/pom.xml]: Please add a comment to replace this with jackson-modules-java8 from jackson 2.8.5

**Comment by gjoranv** [comment]
You must add the new bundle to container-disc/pom.xml preinstall list.

**Comment by bjorncs** [comment]
I will address all comments in new PR.

**Pull request merged**